placement/placement.go

```diff
@@ -533,8 +550,24 @@ func IsInstanceLeaving(instance services.PlacementInstance) bool {
 	return !newInstance
 }
 
-// ByIDAscending sorts PlacementInstance by ID ascending
-type ByIDAscending []services.PlacementInstance
+// Instances is a slice of instances that can produce a debug string.
+type Instances []Instance
+
+func (i Instances) String() string {
+	if len(i) == 0 {
+		return "[]"
+	}
+	var strs []string
```

Comment:
nit: preallocate enough capacity? Also might wanna use a `bytes.Buffer`?

Reply:
bytes.Buffer seems an overkill since this is only used in debugging